packages/core/src/decorators/command/command-options.ts

```diff
@@ -23,4 +25,9 @@ export interface CommandOptions {
    * Set default permission
    */
   defaultPermission?: boolean;
+
+  /**
+   * Command Type (Slash or Context Menu)
+   */
+  type: ApplicationCommandTypes;
```

Comment:
Might be worth making it optional and adding a slash to the commands by default?

Reply:
I actually realized I forgot to make it optional shortly after opening this merge request, and fixed it but forgot to push the commit! I made this property optional and defaulted it to a slash command in `BuildApplicationCommandService`.